src/scripts.js

```diff
@@ -93,13 +101,18 @@ function hideAllForms() {
   hideElement(hydrationForm);
   hideElement(sleepForm);
   hideElement(activityForm);
+  hideElement(formSuccessNotification);
+  hideElement(formErrorNotification);
 }
```

Comment:
I wonder if we added these into an array if we could make this shorter  something like 

let elements = [hydrationForm,  hydrationForm, activityForm]
elements.forEach(element => hideElement(element))